suites/quincy/rbd/tier-2_rbd_namespace.yaml

```diff
@@ -0,0 +1,85 @@
+# Tier2: Added RBD namespace creation in default pool testing
+# Tier-level: 2
+# Test-Suite: tier-2_rbd_namespace.yaml
+#
+# Ceph RBD component.
```

Comment:
please remove this line it's redundant.

Reply:
:nit: @sunilangadi2 this is just a comment, should be fine, let me know if you disagree